audit/pkg/config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"context"
+
+	"github.com/owncloud/ocis/ocis-pkg/shared"
+)
+
+// Config combines all available configuration parts.
+type Config struct {
+	*shared.Commons
+
+	Service Service
+
+	Log   *Log  `ocisConfig:"log"`
+	Debug Debug `ocisConfig:"debug"`
+
+	Events   Events   `ocisConfig:"events"`
+	Auditlog Auditlog `ocisConfig:"auditlog"`
+
+	Context context.Context
+}
+
+// Events combines the configuration options for the event bus.
+type Events struct {
+	Endpoint      string `ocisConfig:"events_endpoint" env:"AUDIT_EVENTS_ENDPOINT"`
```

Comment:
could you please add description tags? see eg. https://github.com/owncloud/ocis/blob/0dcf9524f06cf1aa639710581ec44a81065e81ab/accounts/pkg/config/config.go#L28

Reply:
done